e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -119,4 +119,37 @@ describe('Messaging', () => {
             });
         });
     });
+    it('M17455 - Focus does not move for non-character keys', () => {
```

Comment:
This is failing due to the opened "Add new members modal" from previous test.  Should do ``cy.visit('/')`` first to start the test (or better do ``cy.visit('/')`` before each test)

Reply:
Yes, failing on me.  Like I mentioned, the "Add new members modal" remains open before this test.

Reply:
Ah! I got confused between the `before()` hook and the `beforeEach()` hook :-) I think this should be good now - thanks for explaining!